app/models/organisation.rb

```diff
@@ -7,4 +7,8 @@ class Organisation < ApplicationRecord
 
   scope :sorted_by_name, -> { order(name: :asc) }
   scope :delivery_partners, -> { sorted_by_name.where(service_owner: false) }
+
+  def is_government_organisation?
```

Comment:
Given the object is of the class `Organisation` and the method is an instance method it will always be called _on_ an organisation. Given this I'd consider chopping the name down to `organisation.is_government?`. Not something that should block this work though!